### PR TITLE
{bio}[fosscuda/2019b] Bonito v0.3.2 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
~~#11749~~ (CuPy)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3309.joltik.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz (cascadelake), Python 3.6.8
See https://gist.github.com/06559ab439d4e7243c7900effead0f04 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**FAILED**
Build succeeded for 0 out of 5 (1 easyconfigs in total)
node152.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/4e1b54d6d29bf6d1075bddfede5cbb35 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
bear-pg0212u15b.bear.cluster - Linux centos linux 8.2.2004, x86_64, Intel(R) Xeon(R) CPU E5-2640 v4 @ 2.40GHz (broadwell), Python 3.6.8
See https://gist.github.com/ff2400a3490dc807aec50cbc74cafbeb for a full test report.

**Comment by branfosj** [comment]
> Test report by @branfosj
> **FAILED**
> Build succeeded for 0 out of 1 (1 easyconfigs in total)
> bear-pg0212u15b.bear.cluster - Linux centos linux 8.2.2004, x86_64, Intel(R) Xeon(R) CPU E5-2640 v4 @ 2.40GHz (broadwell), Python 3.6.8
> See https://gist.github.com/ff2400a3490dc807aec50cbc74cafbeb for a full test report.

During the build of parasail (python library) we attempt to build parasail. This is failing with `libtoolize not found`. Rather than adding a libtool builddep, I think it makes more sense to use a separate parasail install, as we do in the standalone python-parasail installs.

File: easybuild/easyconfigs/b/Bonito/Bonito-0.3.2-fosscuda-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('toml', '0.10.2', {
+        'checksums': ['b3bda1d108d5dd99f4a20d24d9c348e91c4db7ab1b749200bded2f839ccbe68f'],
+    }),
+    ('tqdm', '4.54.1', {
```
Comment (branfosj) [inline easybuild/easyconfigs/b/Bonito/Bonito-0.3.2-fosscuda-2019b-Python-3.7.4.eb:31]: Unless this requires the newer version, we should use `tqdm-4.41.1-GCCcore-8.3.0.eb` instead.

**Review by branfosj — CHANGES_REQUESTED** [review]

**Comment by branfosj** [comment]
I tested the existing Bonito ecs and I hit the same issue. I've put a suggested fix in #11937.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3300.joltik.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz (cascadelake), Python 3.6.8
See https://gist.github.com/9286d5ed138c33abafcdde10fbef7052 for a full test report.

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0212u15b.bear.cluster - Linux centos linux 8.2.2004, x86_64, Intel(R) Xeon(R) CPU E5-2640 v4 @ 2.40GHz (broadwell), Python 3.6.8
See https://gist.github.com/e6a07b2d4ddaf2b74b56c954bc469ca0 for a full test report.

**Comment by branfosj** [comment]
> Test report by @lexming
> **FAILED**
> Build succeeded for 0 out of 5 (1 easyconfigs in total)
> node152.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
> See https://gist.github.com/4e1b54d6d29bf6d1075bddfede5cbb35 for a full test report.

This failed on building dependencies - so I'll merge this.

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node154.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/bdcf9322b3b00e810af7951d47281a91 for a full test report.

**Pull request merged**